knative-operator/deploy/resources/dashboards/grafana-dash-knative-health.yaml

```diff
@@ -94,7 +94,7 @@ data:
                }
             ],
             "thresholds": "",
-            "title": "Knative Status",
+            "title": "Knative Current Status",
```

Comment:
What other kind of status then the _Current_ Status could be possible ?

Reply:
It depends there is a time-line bellow.

Reply:
@rhuss regarding tests is on my todo list. Started working on this here #686